src/busctl/busctl.c

```diff
@@ -476,12 +476,10 @@ static int find_nodes(sd_bus *bus, const char *service, const char *path, Set *p
                                "org.freedesktop.DBus.Introspectable", "Introspect",
                                &error, &reply, "");
         if (r < 0) {
-                if (many)
```

Comment:
if you drop this check you can drop the argument entirely and remove it from the function prototypes

Reply:
Indeed. Dropped now.